app/lib/fields/extra_fields.rb

```diff
@@ -99,6 +99,10 @@ def encode_extra_field(value)
     case value
     when String
       encode_string_extra_field(value)
+    when Integer
+      value.to_s
+    when Array
+      value.map { |v| encode_extra_field(v) }
```

Comment:
Why do we need an array here? Should we allow arrays at all?

Reply:
Yes, extra_fields support multiple selection. See https://github.com/3scale/system/pull/2752